site/content/en/docs/tasks/run/deployment.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Run Deployment"
+linkTitle: "Deployment"
+date: 2024-07-25
+weight: 6
+description: >
+  Run a Deployment as a Kueue-managed job.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management
+capabilities when running Deployments.
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+1. By default, the integration for `v1/pod` is not enabled.
+   Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version)
+   and enable the `pod` integration.
+
+   A configuration for Kueue with enabled pod integration would look like follows:
+   ```yaml
+   apiVersion: config.kueue.x-k8s.io/v1beta1
+   kind: Configuration
+   integrations:
+     frameworks:
+      - "pod"
+     podOptions:
+       # You can change namespaceSelector to define in which 
+       # namespaces kueue will manage the pods.
+       namespaceSelector:
+         matchExpressions:
+         - key: kubernetes.io/metadata.name
+           operator: NotIn
+           values: [ kube-system, kueue-system ]
+       # Kueue uses podSelector to manage pods with particular 
+       # labels. The default podSelector will match all the pods. 
+       podSelector:
+         matchExpressions:
+         - key: kueue-job
+           operator: In
+           values: [ "true", "True", "yes" ]
+   ```
+
+2. Kueue will run webhooks for all created pods if the pod integration is enabled. The webhook namespaceSelector could be 
```

Comment:
Similarly, this seems like a repetition, I would skip it.

Reply:
Yeah, I understand the intention, but I think this is a bit too much detail copy-pasted, imo. 

Maybe in the introduction we could add some intro like:
```
In this section we demonstrate how to support scheduling Deployments in Kueue 
based on the Plain Pod integration, where every Pod from a Deployment is represented
as a single independent Plain Pod. This approach allows independent resource
management for the Pods, and thus scale up and down of the Deployment.
```
Then, in the "Before you begin" section something like: 
```
Follow steps in <link> to learn how to enable the ``v1/pod` integration and how to 
configure it using the `podOptions` field.
```